demo/malloy-demo-composer/src/electron/main.ts

```diff
@@ -126,4 +126,69 @@ async function registerIPC(): Promise<void> {
   ipcMain.handle("post:open_directory", async (_event) => {
     return await getOpenDirectory();
   });
+
+  // Native application menu
+  const template: (Electron.MenuItem | Electron.MenuItemConstructorOptions)[] =
```

Comment:
It seems unnecessary to add the full menu to the license popup (if we need a menu at all) since most of the actions are not relevant to that window.

If we do have a reason for wanting this menu then we should probably refactor it out to a shared location so we don't have to maintain in two places to keep them in sync.